package.json

```diff
@@ -32,11 +32,12 @@
     "eslint": "^3.12.0",
     "eslint-config-google": "^0.7.1",
     "gulp": "^3.9.1",
-    "gulp-connect": "^5.0.0",
+    "gulp-connect": "^5.5.0",
     "mocha": "^3.2.0",
     "sinon": "^1.17.7",
     "sinon-chai": "^2.8.0",
-    "typescript": "^2.0.3"
+    "typescript": "2.9.1-insiders.20180516",
+    "vscode-chrome-debug-core": "3.23.8"
```

Comment:
not sure why this is added.  was it types?

Reply:
I think it's fixed in LH now, so can be dropped 👍

Reply:
hmm, still needed because without that tsc build failed

Reply:
huh. weird. we're about to remove that dependency: https://github.com/GoogleChrome/lighthouse/pull/5836

Reply:
probably we will wait for PR to be merged...

Reply:
dropped 💥 
just waiting for next LH release